net/ddns-scripts/Makefile

```diff
@@ -180,6 +187,7 @@ define Package/ddns-scripts-cnkuai
   $(call Package/ddns-scripts/Default)
   TITLE:=CnKuai API
   DEPENDS:=ddns-scripts +curl +giflib-utils
+  PROVIDES:=ddns-scripts_cnkuai_cn
```

Comment:
- This is not necessary. I could not find this package in OpenWrt 19.07. 
- Also... this provides should be for package, which is below, right?

Reply:
similar to https://github.com/openwrt/packages/pull/19597#discussion_r998089071